scarlet/blend.py

```diff
@@ -8,7 +8,7 @@
 logger = logging.getLogger("scarlet.blend")
 
 
-class Blend(ComponentTree, Scene):
+class Blend(ComponentTree):
```

Comment:
This is now somewhat inconsistent:
* `Blend` is now not derived from `Scene` anymore, but line 32 below will attempt to initialize `self` as `Scene`. I'm surprised that this even works.
* `Observation` is still derived from `Scene`, so now `Blend` and `Observation` behave differently in this regard: the former *has* a scene, the latter *is* a scene.

We have agreed that we would alter this consistently, i.e. both should *have* `MetaData` because that clarifies what we mean by `Scene`. For now, please derive `Blend` from `ComponentTree, Scene` as before.

Reply:
Well, I am happy to revert this back, since apparently I missed the concept, but I actually thought blend as an object wrapped around the models while Scene and Observations are representations of the space in which the data live. That helped me disambiguating between get_model in observations and in Blend.

Reply:
Having just said that, I realise that Scene is the abstract space between the data and the model and as such, it makes sense that Blend and Observations both inherit from Scene. Fine I'll do the change.